src/machines/builder_route/index.js

```diff
@@ -0,0 +1,30 @@
+// eslint-disable-next-line node/no-unpublished-import
+import { Machine } from 'xstate';
+
+import { configuration_config } from './configuration.config';
+import { configuration_options } from './configuration.options';
+import { editor_options } from './editor.options';
+import { editor_config } from './editor.config';
+
+
+export const builder_route_options = {
+	guards: {
+		...configuration_options.guards,
+		...editor_options.guards
+	},
+	actions: {
+		...configuration_options.actions,
+		...editor_options.actions
+	}
+};
+
+export const builder_route_config = {
+	id: 'Builder',
+	type: 'parallel',
+	states: {
+		Configurable: configuration_config,
+		Editable: editor_config,
```

Comment:
Here I'd use `Config` and `Editor` here rather than `Configurable` and `Editable`

Reply:
My aim was to use words that describe state, such as being in an `Editable` state. Let me know if you still prefer `Editor`.

Reply:
I do indeed, as I wouldn't know how to interpret `Editable` in this context (i.e. I wondered what is supposed to be editable)

Reply:
Alright, I'll change it, but just to clarify, it is the builder that's in an `Editable` and `Configurable` state.